brokers/che-plugin-broker/broker.go

```diff
@@ -43,30 +43,23 @@ const (
 	Yaml
 )
 
-// ChePluginBroker is used to process Che plugins
-type ChePluginBroker interface {
```

Comment:
@davidfestal this is something that can affect your work

Reply:
Since they're all the same, you should be able to use `common.BrokerImpl` instead.